sparkdl/horovod/runner_base.py

```diff
@@ -35,7 +35,8 @@ class HorovodRunnerBase(object):
     .. note:: Horovod is a distributed training framework developed by Uber.
     """
 
-    def __init__(self, *, np, driver_log_verbosity="all"):  # pylint: disable=invalid-name
+    # pylint: disable=invalid-name
+    def __init__(self, *, np, driver_log_verbosity="log_callback_only"):
```

Comment:
Note: In next release the default log level is `log_callback_only`, see
https://github.com/databricks/spark-deep-learning-private/pull/263